src/app/api/rpc/[fn]/route.ts

```
@@ -93,95 +96,108 @@ export async function POST(req: NextRequest, context: { params: Promise<{ fn: st
+    const rawRole = sessionUser.role ?? ""
+    const role = typeof rawRole === "string" ? rawRole : String(rawRole)
+    const donVi = sessionUser.don_vi != null ? String(sessionUser.don_vi) : ""
+    const diaBan = sessionUser.dia_ban_id != null ? String(sessionUser.dia_ban_id) : ""
+    const khoaPhong = sessionUser.khoa_phong != null ? String(sessionUser.khoa_phong) : ""
+    const userId = sessionUser.id != null ? String(sessionUser.id) : ""
+    const appRole = toAppRoleClaim(role)
+    // (debug removed)
 
     // Sanitize tenant parameter for non-global users to enforce isolation
     // EXCEPTION: regional_leader users can see multiple tenants, don't override p_don_vi
     let body: Record<string, unknown> =
```

ESLint

[error] 'body' is never reassigned. Use 'const' instead.

(prefer-const)